notebooks/community/sdk/sdk_pytorch_lightning_custom_container_training.ipynb

```diff
@@ -0,0 +1,1562 @@
+{
```

Comment:
<p>For gcsfuse conversion, you need to make sure all the subdirs exist or created. Like:</p><p> </p><p>gs_prefix = 'gs://'</p><p>gcsfuse_prefix = '/gcs/'</p><p>if args.model_dir.startswith(gs_prefix):</p><p>&nbsp;&nbsp;args.model_dir = args.model_dir.replace(gs_prefix, gcsfuse_prefix)</p><p>&nbsp;&nbsp;dirpath = os.path.split(args.model_dir)[0]</p><p>&nbsp;&nbsp;if not os.path.isdir(dirpath):</p><p>&nbsp;&nbsp;&nbsp;&nbsp;os.makedirs(dirpath)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/263/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>